worker/executor.go

```diff
@@ -109,7 +109,7 @@ func (e *executor) channelID(edge *pb.DirectedEdge) int {
 	b := make([]byte, len(attr)+8)
```

Comment:
Remove hard-coded "8".
`b := make([]byte, len(attr)+ unsafe.Sizeof(uid)`

Reply:
No need to use `unsafe.Sizeof(uid)` as uint64 size is always 8.